README.md

```diff
@@ -1 +1,37 @@
-# hw01
+# Computing for the Social Sciences
+## The First Homework
+
+#### Who Am I? (... I am Jean Valjean!??)
+
+![Here I Am](Dhruv_Headshot.jpeg)
+
+I'm Jesse Orr, a second year student at Harris from **Baltimore, MD**.  Well, technically **Owings Mills, MD**, a suburb of Baltimore.
+
+After finishing high school in Maryland, I first came to Chicago in 2011 when I began my UChicago career as an undergrad, where I lived in *Crown House* in South Campus.
+
+I knew that I wanted to learn a foreign language, and I already had taken Spanish all through school (plus, spoke it as a heritage speaker) so I began to take **Hindi**.  This would lead to all sorts of interesting life outcomes, and would keep me in the UChicago orbit...
+
+As an undergrad, I was engaged in a variety of different activities, namely:
+
+1. ChoMUN
+2. Team
+3. MUNUC
+4. ...
+5. ...
+6. ...
+7. I wrote one or two *Maroon* articles?
```

Comment:
Please try to use an unordered list as well (next time when you submit other assignments or when you use the markdown.)